pom.xml

```diff
@@ -1644,11 +1644,15 @@
                                 <!-- XXX: Inline and drop the version
                                 declaration once properly supported. See
                                 https://youtrack.jetbrains.com/issue/IDEA-342187. -->
+                                <!-- XXX: This dependency is incompatible with
+                                Error Prone 2.38.0+. Re-enabled it once it is
+                                compatible again.
                                 <path>
                                     <groupId>com.github.lhotari</groupId>
                                     <artifactId>reactor-error-prone</artifactId>
                                     <version>${version.reactor-error-prone}</version>
                                 </path>
+                                -->
```

Comment:
Decided to drop this in #1650. PRs can be merged in either order; I'll resolve the conflict.